### ISSUE TITLE
Update field on create marker

### ISSUE THREAD

**Issue by fizzi** [Budibase/budibase#8037]
### Discussed in https://github.com/Budibase/budibase/discussions/8021

<div type='discussions-op-text'>

<sup>Originally posted by **fizzi** September 28, 2022</sup>
Hi all, 

i am trying to update a field when a marker is created on the map. I defined 2 actions which updated two text fields. I also tried with numbers.

![Schermata 2022-09-28 alle 15 44 56](https://user-images.githubusercontent.com/444291/192795330-326d9ef9-e308-478f-b7df-f78764125033.png)

![Schermata 2022-09-28 alle 15 45 11](https://user-images.githubusercontent.com/444291/192795388-c4a98e96-a22f-48d2-a629-91d9608d3de7.png)

When I create the marker on the map, no fields are updated.

Am I wrong something?

Thanks,</div>

**Comment by aricooperdavis** [comment]
Agreed, this is a bug that currently breaks actions on clicking map marker icons.
